src/cluster/troubleshooting.rst

```diff
@@ -0,0 +1,114 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _cluster/troubleshooting:
+
+=============================================
+Troubleshooting CouchDB 3 with Weather Report
+=============================================
+
+.. _cluster/troubleshooting/overview:
+
+Overview
+========
+
+``weatherreport`` is an escript and set of tools that diagnoses common
+problems which could affect a CouchDB version 3 node or cluster. It
+does not support version 4 or later.
```

Comment:
Should we explicitly mention (lack of) support for version 2 and earlier, as well?

Reply:
We're [importing weatherreport into the 3.x branch](https://github.com/apache/couchdb/pull/3312), so it's not clear how someone could use it in version 2 without significant effort. But, if they _did_ make that effort, most of the checks would likely work. So while it could mostly be used with 2, I don't know if we really want to mention it?